### PR TITLE
Remove obsolete links & update screenshot.

### LINKED ISSUE #1494: Incorrect URL for live seed-element page
The URL located in `1.0/docs/start/reusableelements.md` for the live version of the seed-element page is incorrect. I am submitting a PR with the correct URL.

### PR DESCRIPTION
**Pull request by arthurevans**
Fixes #1494. Remove two obsolete links & add an updated screenshot for the landing page.

### CODE REVIEW

**Comment by robdodson** [comment]
so many white space changes :( Not sure what the actual change was?

File: 1.0/docs/start/reusableelements.md [hunk 1]
```diff
@@ -57,21 +57,21 @@ will need some extra configuration. See
...
-When importing other components, follow the examples in `<seed-element>` and write 
+When importing other components, follow the examples in `<seed-element>` and write
 the paths as if the component folders were siblings to your `test-element` folder.
 
     <link rel="import" href="../paper-button/paper-button.html">
 
-A good way to quickly sanity 
-test your component is to access your demo.html file using polyserve. There are several ways to do this
+A good way to quickly sanity
+test your component is to access your demo file using polyserve. There are several ways to do this
```
Comment (arthurevans) [inline 1.0/docs/start/reusableelements.md]: Here.

File: 1.0/docs/start/reusableelements.md [hunk 2]
```diff
@@ -261,29 +260,24 @@ You can now share the link to your element hosted on GitHub pages with the
 world. As we used the `seed-element` repo, {{site.project_title}} will give you
 a styled component page out of the box that looks a little like this:
 
-![Preview of the component langing page, displaying the element title in the header with a demo link next to it. The rest of the page contains formatted summary and attribute/method/event information parsed from the documentation in your element](/images/publishing-polymer-elements/image_6.png)
+![Preview of the component langing page, displaying the element title in the header with a demo link next to it. The rest of the page contains formatted summary and attribute/method/event information parsed from the documentation in your element](../../images/reusableelements/documentation-page.png)
```
Comment (arthurevans) [inline 1.0/docs/start/reusableelements.md]: Here.

File: 1.0/docs/start/reusableelements.md [hunk 3]
```diff
@@ -261,29 +260,24 @@ You can now share the link to your element hosted on GitHub pages with the
...
 
-You can check out the [live](http://polymerlabs.github.io/seed-
-element/components/seed-element/) version of this page for the `seed-element`
-project.
```
Comment (arthurevans) [inline 1.0/docs/start/reusableelements.md]: Here.

File: 1.0/docs/start/reusableelements.md [hunk 4]
```diff
@@ -200,34 +200,33 @@ You should now be able to install your component. Change to a new directory for
 Now you have a tagged release for your element. Next up, create a
 meaningful demo and landing page for your element.
 
-While optional, we recommend modifying the `demo/index.html` page to provide a real-
-world [example](http://googlewebcomponents.github.io/google-chart/components
-/google-chart/demo.html) of your element. Developers can get a feel for how your
+While optional, we recommend modifying the `demo/index.html` page to provide a
+real-world example of your element.  Developers can get a feel for how your
```
Comment (arthurevans) [inline 1.0/docs/start/reusableelements.md]: Here (removed obsolete link).

**Comment by arthurevans** [comment]
Sorry for the WS changes--but they're all in the right direction!
Marked the actual changes.

**Comment by robdodson** [comment]
LGTM thanks!

**Pull request merged**